c10/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ endif()
 if(C10_USE_GLOG)
     target_link_libraries(c10 PUBLIC glog::glog)
 endif()
-target_link_libraries(c10 PRIVATE fmt::fmt-header-only)
+target_link_libraries(c10 PUBLIC fmt::fmt-header-only)
```

Comment:
Do we really have to? We have fmt formatting for other things and we never needed that right?

Reply:
I cannot `#include <fmt/format.h>` without this. It is very awkward.

There are calls to fmt/format.h functions in c10, but no custom formatting (which is a template specialization). I'm not sure if this is at all resolvable without putting the template specialization where SymInt is defined.

Reply:
This leads to compilation errors I'm afraid :/

Reply:
yeah I give up lol